components/top-nav/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@govuk-react/top-nav",
+  "version": "0.1.27",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@govuk-react/button": "^0.1.27",
+    "@govuk-react/constants": "^0.1.26",
+    "@govuk-react/icons": "^0.1.27",
+    "@govuk-react/search-box": "^0.1.27",
+    "govuk-colours": "^1.0.3"
+  },
+  "peerDependencies": {
+    "glamorous": ">=4",
+    "prop-types": ">=15",
+    "react": ">=16.2"
```

Comment:
need to update peerDependency of react to 16.2 in packages/govuk-react too, probably worth doing it everywhere with a find replace

Reply:
done.